### PR TITLE
Update DSTU2 documentation for Conformance with new authorize token endpoints.

### PR DESCRIPTION
**Pull request by sravan559**

### CODE REVIEW

File: lib/dstu2_resources.rb [hunk 1]
```diff
@@ -1300,22 +1373,22 @@ module Resources
                 {
                   "name": "patient",
                   "type": "reference",
-                  "documentation": "The identity of a patient to list dispenses for. It is a required field"
+                  "documentation": "The identity of a patient to list medication orders for. It is a required field"
```
Comment (joykoduru) [inline lib/dstu2_resources.rb]: Nit: End line with a period for consistency and I think that is the standard.

Comment (MaxPhilips) [inline lib/dstu2_resources.rb]: This data is just copied over from our conformance statement, so we should add the period in our actual server before adding it here.  No period is fine given our current conformance

Comment (sravan559) [inline lib/dstu2_resources.rb]: @MaxPhilips  👍

**Comment by MaxPhilips** [comment]
+1

**Comment by jfrhong** [comment]
+1

**Pull request merged**